plugin/azure/README.md

```diff
@@ -0,0 +1,52 @@
+# azure
+
+## Name
+
+*azure* - enables serving zone data from Microsoft Azure DNS service.
+
+## Description
+
+The azure plugin is useful for serving zones from resource record
+sets in Microsoft Azure DNS. The `azure` plugin can be used when coredns is deployed on Azure or elsewhere.
+This plugin supports all the DNS records supported by Azure, viz. A, AAAA, CAA, CNAME, MX, NS, PTR, SOA, SRV, and TXT record types.
+
+## Syntax
+
+~~~ txt
+azure resource-group:dns-zone {
```

Comment:
please fix this readme (again)

Reply:
Sorry for the back and forth on the README. 
I am not sure about this particular case though, what changes do are you pointing at here?